cms/static/js/spec/views/previous_video_upload_spec.js

```diff
@@ -14,7 +14,21 @@ define(
                     },
                     view = new PreviousVideoUploadView({
                         model: new Backbone.Model($.extend({}, defaultData, modelData)),
-                        videoHandlerUrl: '/videos/course-v1:org.0+course_0+Run_0'
+                        videoHandlerUrl: '/videos/course-v1:org.0+course_0+Run_0',
+                        videoImageSettings: {
+                            max_size: 2 * 1024 * 1024,
+                            min_size: 2 * 1024,
+                            max_width: 1280,
+                            max_height: 720,
+                            supported_file_formats: {
+                                '.png': 'image/png',
+                                '.gif': 'image/gif',
+                                '.bmp': 'image/bmp',
+                                '.bmp2': 'image/x-ms-bmp',   // PIL gives x-ms-bmp format
+                                '.jpg': 'image/jpeg',
+                                '.jpeg': 'image/jpeg'
+                            }
```

Comment:
Why we need to set all these values to videoImageSettings? Can't we set it to empty? These settings are only needed and tested in `video_thumbnail_spec.js` so why we need these here?